.azure-pipelines/client.test.live.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       - script: npm run live-test-client-$(Package)-node -- -- --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- $(AdditionalTestArguments)
         displayName: 'npm run live-test-client-$(Package)-node'
         env:
+          DEBUG: $(DEBUG)
```

Comment:
How will this work end-to-end?  When queuing a build, the `DEBUG` build variable can be set to some value, which will be passed through to the tests?  What's an example scenario where someone would use this?

Reply:
@jeremymeng is probably best equipped to answer this question as he rasied the issue in the Teams discussion. 

He requested that we put support for `DEBUG` back in for Event Hubs and Service Bus. Those projects' READMEs reference using `DEBUG`. (example: https://github.com/Azure/azure-sdk-for-js/blob/ac40dcc9438c652bf76724c7b5ff7ba108d11825/sdk/eventhub/event-hubs/README.md#debug-logs)

Reply:
Does it have to be passed explicitly? Variable set at queue time should be available in the environment variables for the pipeline. I believed it worked for me without having to do so.  The `env` block probably allows setting it to a variable with different name.

Reply:
I believe @jeremymeng is correct (for non-secret variables).  So maybe no YAML changes are required here?

Reply:
Indeed, @jeremymeng is right. 

* Variables set at queue time are available as environment variables to steps.
* The variables set at queue time also don't need to be set in the Variables section of a DevOps pipeline in order for the above behavior to apply. I just verified an example of this in a playground pipeline. This means we can keep the `DEBUG` out of the Variables section.

Reply:
I added it in the section to make it easier to use at queue time. i.e., don't need to click the + button, type DEBUG then type true. Though it's not a commonly used scenario so I am fine with removing it.